### DISCUSSION TITLE
Welcome to solo2 Discussions!

### DISCUSSION THREAD

**Discussion by nickray** [solokeys/solo2#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Please note**: This repository is for _technical_ discussion of the _firmware_. For discussions regarding crowdfunding campaigns or the user-level perspective in general, please use https://github.com/solokeys/kickstarter2021/discussions.

To get started with building the firmware: https://hackmd.io/@solokeys/solo2-getting-started

**Comment by sparks79** [comment]
I have just received my first Solokey and I am Using Windows 10.
I can't get it to work.
Looking around the Net people say I have to Register it.
But All the references to that are assuming you are on Linux.
So How do I register it in Windows.
Also I see mention that you can use it on any Site With or Without 2FA.
But I have not seen Any Site offer to Use a Key in place of a Password.
Another thing, if I can succeed in Setting Up Google for 2FA, How would my Wife Log into Google if She Doesn't have a Solokey, would Google be able to Identify Her and Accept her Existing P/W or will she have to have a Solokey like me.
Can you set up google for the Solokey Without having to Use 2FA.
I hope some kind person can help me here

**Reply by conorpp** [reply]
@sparks79 this is the discussion for the Solo 2 campaign.  For Solo "v1" support, please reach out to hello@solokeys.com.  For most sites you can use Solo as a 2FA token.  To log into windows with a Solo, this typically only works when Windows is part of a business setup like Azure Active Directory.  Regarding your wife logging in -- she can have her own Solo and log in it with.  Accounts can have multiple Solos registered.
